src/liveMarkbookCalculator.js

```diff
@@ -137,14 +144,96 @@ const parseMarkbook = () => {
+            }
+        });
+    });
+};
+
+/**
+ * @desc Iterates over the current markbook and highlights changed cells
+ */
+const highlightChanges = () => {
+    $('#markbookTable table tbody > tr:gt(0)').each(function (i) {
+        const row = $(this);
+
+        const currentMark = row.find('td:nth-child(2) > input');
+        if (currentMark.val() !== initialMarkbook[i].mark.val) {
+            currentMark.parent().css('background-color', '#ffe499');
+        } else {
+            currentMark.parent().css('background-color', initialMarkbook[i].mark.bgColor);
+        }
+
+        const currentWeight = row.find('td:nth-child(4) > input');
+        if (currentWeight.val() !== initialMarkbook[i].weight.val) {
+            currentWeight.parent().css('background-color', '#ffe499');
+        } else {
+            currentWeight.parent().css('background-color', initialMarkbook[i].weight.bgColor);
+        }
+
+        const currentDenominator = row.find('td:nth-child(5) > input');
+        if (currentDenominator.val() !== initialMarkbook[i].denominator.val) {
+            currentDenominator.parent().css('background-color', '#ffe499');
+        } else {
+            currentDenominator.parent().css('background-color', initialMarkbook[i].denominator.bgColor);
+        }
+    });
+};
+
 /**
  * @desc Converts the current open markbook to an editable format
  */
 const makeMarkbookEditable = () => {
-    $('#markbookTable table').prepend('<style type="text/css">input[type="number"]::-webkit-outer-spin-button,input[type="number"]::-webkit-inner-spin-button {-webkit-appearance: none;margin: 0;} input[type="number"] {-moz-appearance: textfield; margin: 0; border: none; display: inline; font-family: Monaco, Courier, monospace; font-size: inherit; padding: 0; text-align: center; width: 30pt; background-color: inherit;}</style>');
+    $('#markbookTable table').prepend(`
+    <style type="text/css">
+        input[type="number"]::-webkit-outer-spin-button, input[type="number"]::-webkit-inner-spin-button {
+            -webkit-appearance: none;
+            margin: 0;
+        } 
+        
+        input[type="number"] {
+            -moz-appearance: textfield; 
+            margin: 0; 
+            border: none; 
+            display: inline; 
+            font-family: Monaco, Courier, monospace; 
+            font-size: inherit; 
+            padding: 0; 
+            text-align: center; 
+            width: 30pt; 
+            background-color: inherit;
+        }
+    </style>`);
     initialFinalMark = parseFloat($('#markbookTable > div > div').text().substr(11)); // Grab everything after 'Term Mark: '
     $('#markbookTable table tbody td:nth-child(n+2):nth-child(-n+5):not(:nth-child(3))').each(function () {
-        const value = $(this).text();
+        let value = $(this).text();
+
+        if (!isNaN(parseFloat(value)) && value !== '') {
+            value = +parseFloat(value).toFixed(2);
```

Comment:
This will very likely cause an edge case where the rounding causes a calculation difference and making the calculated `finalMark` off by a few decimal points since the backend calculation that Maplewood uses has infinite precision for each mark.

Reply:
This is inevitable. Rather than rounding the initial values to match the rounding of the final value, the only alternative would be to round to 3 decimal places when calculating the final, then compare with the initial. However, even with this approach, the infinite precision of Maplewood can still cause an issue.

Reply:
Since on line 32, the mark is already rounded to the nearest 2 decimal places
```
row.find('td:nth-child(2) > input').val(+mark.toFixed(2));
```
This commit shouldn't cause too much issue.